packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -54,12 +54,12 @@ export interface CookieBannerProps {
     locale: CookieBannerLocale;
 
     /**
-     * Allows consumers to pass in a specific cookie statement link.
+     * Allows a url to be passed for the cookie statement link.
```

Comment:
"Allows" might imply this is optional. Could it be worth rewording to something like:

"The URL of the cookie statement page the banner should link to."? 

Same for the link prop below too